### ISSUE TITLE
Add support for calendar time

### ISSUE THREAD

**Issue by evanstoner** [stefanpenner/ember-moment#126]
This is another Moment display function, similar to `fromNow()`.

http://momentjs.com/docs/#/displaying/calendar-time/

**Comment by bschnelle** [comment]
+1

**Comment by Amareis** [comment]
+1. It's very good function and it must be supported, IMHO.

**Comment by jasonmit** [comment]
My newest PR (#139) will make adding new features like this easier through composition.  Just waiting on @stefanpenner for a review and then will follow up with a PR to close this out.

**Comment by jasonmit** [comment]
Versioned as 5.1.0, I will create a separate issue for adding the computed property macro for it
